libs/testing/fixtures/src/lib/verificationCode.ts

```diff
@@ -1,13 +1,8 @@
-import faker from 'faker'
+import { faker } from '@faker-js/faker'
 
 /**
  * Creates a random `length` digit verification code as a string
  * @param length Controls the number of digit in the verification code
  */
 export const createVerificationCode = (length = 3) =>
-  faker.datatype
-    .number({
-      min: Math.pow(10, length - 1),
-      max: Math.pow(10, length) - 1,
-    })
-    .toString()
+  faker.string.numeric(length).toString()
```

Comment:
`toString()` is redundant here.

```suggestion
  faker.string.numeric(length)
```

Reply:
Cood catch, thank you.